substrate/client/network/test/src/sync.rs

```diff
@@ -1249,6 +1251,35 @@ async fn warp_sync() {
 	.await;
 }
 
+/// If there is a finalized state in the DB, warp sync falls back to full sync.
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
```

Comment:
Doesn't work with the default config?

Reply:
To be honest, I just put the same config as in other tests in this file to be consistent. It does work with the single-threaded runtime though.